openupgradelib/openupgrade_merge_records.py

```diff
@@ -553,18 +582,18 @@ def _adjust_merged_values_orm(
         if not field.store or field.compute or field.related:
             continue  # don't do anything on these cases
         op = field_spec.get(field.name, False)
-        if field.type != "reference":
-            _list = all_records.mapped(field.name)
-        else:
+        if field.type == "reference":
```

Comment:
This changes is not needed.

Reply:
I know, it was just changing the order. It bothers you? I think it's better doing this way, because it's more readable listing the specific case in the `if` and the rest in the `else`. If in the future we need to add anther case, a simple `elif` would only be added.

Reply:
Readability is debatable. I think it's the same, but the main principle is to reduce the diff on unrelated things. This is totally unrelated to the goal of the PR. You can do such refactoring in other PR labeling it `[REF]`.

Reply:
Done